cmd/oras/push.go

```diff
@@ -73,6 +74,10 @@ Example - Push file "hi.txt" with config type "application/vnd.me.config":
 Example - Push file "hi.txt" with the custom manifest config "config.json" of the custom media type "application/vnd.me.config":
   oras push --config config.json:application/vnd.me.config localhost:5000/hello:latest hi.txt
 
+Example - Push file "hi.txt" and enforce packed manifest type:
```

Comment:
Is it more straightforward and clear as follows?
```suggestion
Example - Push file "hi.txt" and enforce building a specific manifest type:
```